app/src/main/res/layout/profile_chooser_add_view.xml

```diff
@@ -2,29 +2,80 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
 
-  <LinearLayout
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_marginBottom="24dp"
-    android:gravity="center_horizontal"
-    android:orientation="vertical">
-
-    <de.hdodenhof.circleimageview.CircleImageView
-      android:id="@+id/profile_add_button"
-      android:layout_width="72dp"
-      android:layout_height="72dp"
-      android:layout_marginBottom="12dp"
-      android:src="@drawable/ic_add_profile"
-      app:civ_border_color="@color/avatarBorder"
-      app:civ_border_width="1dp" />
-
-    <TextView
-      android:id="@+id/add_profile_text"
-      android:layout_width="wrap_content"
+  <RelativeLayout
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
```

Comment:
The visibility of these layouts should be controlled by a ViewModel instead of kotlin code. Check implementation of ViewModels in other activities.

Reply:
PTAL:)